src/main/java/io/supertokens/webserver/api/jwt/JWKAPI.java

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
So this was probably my OCD kicking in but JWKs is just plural, since the API is to fetch a JWK (even though its an array only one is used) for validation `JWKAPI` seemed like a better name. Im agnostic though so I dont mind either.

Reply:
Done